src/components/DataController/DataController.js

```diff
@@ -172,6 +177,14 @@ var DataController = React.createClass({
               <Col lg={4} lgPush={8} md={6} mdPush={6} sm={6} smPush={6}>
                 <Selector label={"Dataset"} onChange={this.updateAnnCycleDataset} items={ids} />
               </Col>
+              <Col lg={4} lgPush={1} md={6} mdPush={1} sm={6} smPush={1}>
+                <div>
+                  <ControlLabel>Download Data</ControlLabel>
+                  <br/>
```

Comment:
`<br/>` is generally considered undesirable markup. It's not semantic and it forces layout information into what should be content specification only (IOW, it doesn't say what, it says how). If two things need a break between them, there are some options. The simplest ones in this case would be one of the following:

- Put `ControlLabel` and/or the `Button` elements into their own `div`s . That is more properly semantic, though you are relying on the standard styling of div to mean "vertical layout".
- In the CSS, select this ControlLabel (adding a class to it if needed) and style it `display: block`.